modules/subchannel/doc/content/source/postprocessors/SCMDuctHeatPostprocessor.md

```diff
@@ -0,0 +1,19 @@
+# SCMDuctHeatPostprocessor
```

Comment:
preferences between Heat, HeatFlow, HeatRate or HeatFlowRate

the problem with heat is that it technically is energy so wrong unit there